public/js/app/util/__test__/helpers.test.js

```diff
@@ -5,58 +5,42 @@ const coursesTypeOne = {
   course_SV: {
     courseCode: 'ML1616',
     gradeScaleCode: 'AF',
-    title: { sv: 'Industriell projektledning och projektstyrning' },
-    credits: 7.5,
+    title: 'Industriell projektledning och projektstyrning',
+    credits: { formattedWithUnit: '7.5 hp' },
     creditUnitLabel: 'Högskolepoäng',
     creditUnitAbbr: 'hp',
     courseLiterature: '<p>Meddelas vid kursstart.</p>',
   },
   course_EN: {
     courseCode: 'DD1321',
     gradeScaleCode: 'AF',
-    title: { en: 'Applied Programming and Computer Science' },
-    credits: 7.5,
+    title: 'Applied Programming and Computer Science',
+    credits: { formattedWithUnit: '7.5 credits' },
     creditUnitLabel: 'Credits',
     creditUnitAbbr: 'hp',
   },
 }
-const courseTypeTwo = {
-  course: {
-    title: {
-      sv: 'Tillämpad programmering och datalogi',
-      en: 'Applied Programming and Computer Science',
-    },
-    credits: 9,
-    creditUnitLabel: {
-      sv: 'Högskolepoäng',
-      en: 'Credits',
-    },
-    creditUnitAbbr: {
-      sv: 'hp',
-      en: 'hp',
-    },
-  },
-}
+
 describe('combine course name', () => {
   test('combine course name for an object from a detailed information endpoint in Swedish', () => {
-    const courseTitle = combinedCourseName('ML1616', coursesTypeOne.course_SV, 'sv')
-    expect(courseTitle).toBe('ML1616 Industriell projektledning och projektstyrning 7,5 hp')
+    const courseTitle = combinedCourseName('ML1616', coursesTypeOne.course_SV)
+    expect(courseTitle).toBe('ML1616 Industriell projektledning och projektstyrning 7.5 hp')
   })
 
   test('combine course name for an object from a detailed information endpoint in English', () => {
-    const courseTitle = combinedCourseName('ML1616', coursesTypeOne.course_EN, 'en')
+    const courseTitle = combinedCourseName('ML1616', coursesTypeOne.course_EN)
     expect(courseTitle).toBe('ML1616 Applied Programming and Computer Science 7.5 credits')
   })
 
-  test('combine course name for an object from a course rounds information endpoint in Swedish', () => {
-    const courseTitle = combinedCourseName('ALLLLL', courseTypeTwo.course, 'sv')
-    expect(courseTitle).toBe('ALLLLL Tillämpad programmering och datalogi 9.0 hp')
-  })
+  // test('combine course name for an object from a course rounds information endpoint in Swedish', () => {
```

Comment:
.

Reply:
I believe that is a change from this [PR](https://github.com/KTH/kurs-pm-data-admin-web/pull/394#issue-comment-box), so it's probably better if I fix it there first. I branched out from that one because it had changes that I needed in order to run the app at all